### ISSUE TITLE
App Signer Won't Sign Apps

### ISSUE THREAD

**Issue by Brak01** [DanTheMan827/ios-app-signer#40]
When I click start it starts to process the ipa, but then I get the following error:

ERROR VERIFYING CODE SIGNATURE!
/var/folders/w7/h7_kf9191ts1bjth93rl7y700000gn/T/com.DanTheMan827.AppSigner.attQNDYW/out/Payload/Test App.app: invalid Info.plist (plist or signature have been modified)
In architecture: armv7

I am even trying this on an app I copied from my phone and running it without doing any modifications to it, but still getting the same error.

Any advise how to resolve this issue?

**Comment by cmsimike** [comment]
Did you upgrade to the latest xcode today?

**Comment by Brak01** [comment]
No, but I will right now.

Mike Megally wrote:

**Comment by Brak01** [comment]
While I'm waiting for it to download and install..... what will the update, in regards to the iOS App Signer, fix?

**Comment by cmsimike** [comment]
No, I was more curious about whether the new version broke it or not, which isn't the case.

**Comment by Brak01** [comment]
My OS is macOS Sierra version 10.12.1
I just got this computer (this is my first mac), and last week downloaded and installed x-code 8.1 and iOS App Signer 1.8.3. I am wanting to change the icons and display names of some iphone apps.

What version of x-code and ios app signer should I be using?

**Comment by Brak01** [comment]
Should I try a different OS, like maybe el capitan?

**Comment by DanTheMan827** [comment]
I've just tested 1.8.3 on Mac OS 10.12.2 and everything appears to be working fine.

Are you selecting the correct certificate / profile combination?
